wrappers/services.go

```diff
@@ -227,7 +221,7 @@ type StartServicesFlags struct {
 // StartServices starts service units for the applications from the snap which
 // are services. Service units will be started in the order provided by the
 // caller.
-func StartServices(apps []*snap.AppInfo, disabledSvcs []string, flags *StartServicesFlags, inter Interacter, tm timings.Measurer) (err error) {
+func StartServices(apps []*snap.AppInfo, disabledSvcs map[int][]string, flags *StartServicesFlags, inter Interacter, tm timings.Measurer) (err error) {
```

Comment:
doc comment should explain disabledServices? do I understand correctly that the key is a uid? and we will use uid=0 both for system services and for user services of root? shouldn't we have a struct instead and separate those better?

Reply:
Yea I think that's a great suggestion, will change this